html-templates/cbl/student-dashboard.tpl

```diff
@@ -22,24 +23,42 @@
                 </tr>
             </thead>
 
-            <tbody></tbody>
+            <tbody>
+                <tr>
+                    <td class="scoring-domain-col">
+                        <span class="domain-skill">Cite Evidence</span>
+                        <div class="meta">
+                            <span class="domain-competency">Reading Literature, </span>
+                            <span class="domain-teacher">Mr. Smith</span>
+                        </div>
+                    </td>
+                    <td class="level-col"><div class="level-color cbl-level-10">10</div></td>
+                </tr>
+                <tr>
+                    <td class="scoring-domain-col">
+                        <span class="domain-skill">Customize the presentation for the specific purpose, context, and audience</span>
+                        <div class="meta">
+                            <span class="domain-competency">Writing Evidence-based Arguments, </span>
+                            <span class="domain-teacher">Mr. Smith</span>
+                        </div>
+                    </td>
+                    <td class="level-col"><div class="level-color cbl-level-10">10</div></td>
+                </tr>
+            </tbody>
         </table>
     </aside>
-    *}
 
     <header class="page-header">
         <h2 class="header-title">{$Student->FullName|escape}</h2>
 
-        {*
         <div class="cbl-labeled-meter">
-            <h3 class="cbl-meter-label">Graduation Progress</h3>
+            <h3 class="cbl-meter-label level-color">Graduation Progress</h3>
             <div class="cbl-progress-meter cbl-level-10">
                 <div class="cbl-progress-bar" style="width:40%"></div>
                 <div class="cbl-progress-level">L10</div>
                 <div class="cbl-progress-percent">40%</div>
```

Comment:
@ckunkel can you verify that the "graduation progress" feature should remain on-hold for the time being?